tables/vpc_flow_log/vpc_flow_log_table.go

```diff
@@ -54,56 +48,64 @@ func (c *VpcFlowLogTable) EnrichRow(row *VpcFlowLog, sourceEnrichmentFields sche
 
 	// Record standardization
 	row.TpID = xid.New().String()
-
-	// TODO is source type actually the source, i.e compressed file source etc>
-	// should these all be filled in by the source???
-	row.TpSourceType = c.Identifier()
-	//row.TpSourceName = ???
-	//row.TpSourceLocation = ???
 	row.TpIngestTimestamp = time.Now()
 
-	// Hive fields
-	if row.AccountID != nil {
-		row.TpIndex = *row.AccountID
-	} else {
-		row.TpIndex = "default"
-	}
-
-	// TODO: How to handle if the log don't have timestamp value
-	// populate the year, month, day from start time
-	if row.Timestamp != nil {
-		// convert to date in format yy-mm-dd
-		row.TpDate = row.Timestamp.In(time.UTC).Truncate(24 * time.Hour)
-		row.TpTimestamp = *row.Timestamp
-	} else if row.Start != nil {
-		// convert to date in format yy-mm-dd
-		// TODO is Start unix millis?? if so why do we convert it for TpTimestamp
-		row.TpDate = time.UnixMilli(*row.Start).Truncate(24 * time.Hour)
-
-		//convert from unis seconds to milliseconds
-		row.TpTimestamp = time.Unix(0, int64(*row.Start*1000)*int64(time.Millisecond))
-	} else {
-		// TODO: is it correct to fallback to the current time
-		t := time.Now()
-		row.TpDate = time.UnixMilli(t.UnixMilli()).Truncate(24 * time.Hour)
-		row.TpTimestamp = t
-	}
-
-	//row.TpAkas = ???
-	//row.TpTags = ???
-	//row.TpDomains = ???
-	//row.TpEmails = ???
-	//row.TpUsernames = ???
-
 	// ips
 	if row.SrcAddr != nil {
 		row.TpSourceIP = row.SrcAddr
 		row.TpIps = append(row.TpIps, *row.SrcAddr)
 	}
+	if row.PktSrcAddr != nil {
+		row.TpIps = append(row.TpIps, *row.PktSrcAddr)
+	}
 	if row.DstAddr != nil {
 		row.TpDestinationIP = row.DstAddr
 		row.TpIps = append(row.TpIps, *row.DstAddr)
 	}
+	if row.PktDstAddr != nil {
+		row.TpIps = append(row.TpIps, *row.PktDstAddr)
+	}
+
+	// TpIndex
+	if row.InterfaceID != nil {
+		row.TpIndex = *row.InterfaceID
+	} else if row.VPCID != nil {
+		row.TpIndex = *row.VPCID
+	} else if row.SubnetID != nil {
+		row.TpIndex = *row.SubnetID
+	} else {
+		row.TpIndex = "default"
+	}
+
+	// TpSourceLocation
+	if row.SublocationID != nil {
+		row.TpSourceLocation = row.SublocationID
+	}
```

Comment:
We shouldn't be overriding `tp_source_location` for example for an S3 source this will be the name of the file the log came from.